app/assets/javascripts/cohorts/cohort.js.coffee

```diff
@@ -180,6 +181,21 @@ class App.Cohorts.Cohort
       return 0
     if moment(a, 'MMM DD, YYYY').format('YYYYMMDD') > moment(b, 'MMM DD, YYYY').format('YYYYMMDD') then 1 else -1
 
+  sort_everything_else: (a, b) =>
+    if a == null
+      return -1
+    if b == null
+      return 1
+    if !a.substring || !b.substring
```

Comment:
Use `typeof`, or document that if a variable is not a string, this will return `undefined`?